<launch>
 <group ns="lidar1">
  <node name="lidar01" pkg="bluesea2" type="bluesea2_node" output="screen" >
    <param name="type" value="udp" />
    <param name="lidar_port" value="5000" />
    <param name="lidar_ip" value="192.168.158.91" />
    <param name="group_ip" value="224.1.1.99" />
    <param name="local_port" value="50122" />
    <param name="raw_bytes" value="3"/>
    <param name="frame_id" value="map" />
    <param name="output_360" value="true" />
    <param name="output_scan" value="true" />
    <param name="output_cloud" value="false"/>
    <param name="with_angle_filter" value="false"/>
    <param name="min_angle" value="-2.2"/>
    <param name="max_angle" value="2.2"/>
    <param name="max_dist" value="50.0"/>
    <param name="hard_resample" value="false"/>
    <param name="soft_resample" value="false"/>
    <param name="resample_res" value="0.400"/>
    <param name="unit_is_mm" value="true"/>
    <param name="with_confidence" value="true"/>
    <param name="with_checksum" value="true"/>
  </node>
 </group>
</launch>



